CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ Changelog
 
 ### Chores
 
+- Moved ITN redemption to Daedalus application menu ([PR 2401](https://github.com/input-output-hk/daedalus/pull/2401))
+- Adjusted hover PopOver styles on the first tile in the delegation center screen ([PR 2386](https://github.com/input-output-hk/daedalus/pull/2386))
```

Comment:
@yakovkaravelov why is this here?

Reply:
@tomislavhoracek `isUpdateAvailable` was removed after verifying that it is not used anymore by Nikola.

Reply:
This changelog issue was generated by some git merge issue. Will fix it.